mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -432,16 +432,34 @@
     <!-- Text to prompt users to select the tabs to save in the "select tabs" stepof the collection creator -->
     <string name="create_collection_save_to_collection_empty">Select tabs to save</string>
 
-    <!-- Text to prompt users to select how many tabs they have
-    selected in the "select tabs" stepof the collection creator -->
-    <string name="create_collection_save_to_collection_full">%d tabs selected</string>
+    <plurals name="create_collection_save_to_collection_full_plural">
```

Comment:
`%d` also needs to be explained in comments here. Please make sure it's done throughout the file
(@Pike: Can we use "plurals name" here?)